src/index.html

```diff
@@ -3,107 +3,181 @@
-        <input type="email" id="email" name="email" placeholder="Enter your email address">
-        <fieldset class="submit_btn"><p><button id="beta-btn" class="btn">Sign up</button></p></fieldset>
-      </form>
-
-      <!-- RELATED LINKS -->
-      <header class="tabbed-header tabbed-pad-top">
-        <h3 class="subhead tabbed-heading">Related links</h3>
-      </header>
-
-      <ul class="links-ul">
-        <li>Looking for home buying help or struggling to repay your mortgage?<br />
-          <a class="link-with-icon" href="/find-a-housing-counselor/" target="_blank">Find a HUD-approved housing counselor<span class="cf-icon cf-icon-right"></span></a></li>
-        <li>Have more mortgage questions?<br />
-          <a class="link-with-icon" href="/askcfpb/search/?selected_facets=category_exact:mortgages" target="_blank">Get answers to common questions<span class="cf-icon cf-icon-right"></span></a></li>
-        <li>Have a complaint about your mortgage? <br />
-          <a class="link-with-icon" href="/complaint/#mortgage" target="_blank">Submit a mortgage complaint<span class="cf-icon cf-icon-right"></span></a></li>
-        <li>Wondering about our other mortgage work? <br />
-          <a class="link-with-icon" href="/mortgage/" target="_blank">Learn how CFPB is protecting mortgage borrowers<span class="cf-icon cf-icon-right"></span></a></li>
-      </ul>
+{% block content %}    
+<main class="content content__2-1 content__bleedbar" id="main" role="main">
+
+    <section class="content_hero">
+      
+      <!-- BACKGROUND IMAGE FOR THE HERO -->
+      <div id="hero-bg-image"></div>
+      
+       <!-- HERO -->
+      <div class="content_wrapper content_no-sidebar">
+          <section class="content_main">
+            <h1>Owning a Home</h1>
+            <h2>We're here to help you with your home-buying process. Learn what to expect and what questions to ask, and get tools and resources to help you make decisions.</h2>
+          </section>
+      </div>
+    </section>
+
+    <div class="content_wrapper">
+        <section class="content_main">
+
+          <section class="section__intro">      
+            <div class="content-l_col content-l_col-1-2">
+              <h2>Know the process</h2>
+              <p>Buying a home can be a daunting process. If you're feeling overwhelmed, you're not alone. We've got tools and resources to help you every step of the way.</p>
+            </div>
+
+            <div class="content-l_col content-l_col-1-2">
+              <img src="http://placekitten.com/380/200" alt="Illustration of the home-buying process">
+          </div>
+          </section>
+
+          <section class="section__phase grid_column__top-divider block__flush-top">
+            <figure class="phase_illu content-l_col content-l_col-1-4">
+              <img src="http://placekitten.com/g/150/150" alt="Illustration of the home-buying process">
+            </figure>
+
```

Comment:
I'll bookmark this for now, but I plan to add that pattern to Capital Framework and we can then pull it in here as a dependency. See https://github.com/cfpb/cfgov-refresh/issues/321